distributed/scheduler.py

```diff
@@ -2086,6 +2091,16 @@ def _transition(
             return recommendations, client_msgs, worker_msgs
         except Exception:
             logger.exception("Error transitioning %r from %r to %r", key, start, finish)
+            self.log_event(
+                "transistions",
```

Comment:
Typo:
```suggestion
                "transitions",
```

As a more general note, I think I'd _personally_ prefer grouping messages by more coarse-grained topics similar to how we group your average logs. I.e., in this case I'd personally push this into a `scheduler` topic. We currently lack guidelines, so this is completely up to you. I'm just thinking about how to organize this better (anything is preferred to the `all` topic) and will write up an RFC for internal guidelines at some point.